ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatCommandModal.qml

```diff
@@ -127,28 +139,11 @@ ModalPopup {
                 }
             }
 
-            SVGImage {
-                width: 16
-                height: 16
-                visible: warningText.visible
-                source: "../../../../img/warning.svg"
-                anchors.horizontalCenter: parent.horizontalCenter
-                anchors.bottom: warningText.top
-                anchors.bottomMargin: 4
-            }
-
-            StyledText {
-                id: warningText
-                visible: !root.isRequested
-                //% "You need to request the recipient’s address first.\nAssets won’t be sent yet."
-                text: qsTrId("you-need-to-request-the-recipient-s-address-first--nassets-won-t-be-sent-yet-")
-                color: Style.current.danger
-                horizontalAlignment: Text.AlignHCenter
-                wrapMode: Text.WordWrap
-                anchors.right: parent.right
-                anchors.left: parent.left
+            AddressRequestedValidator {
+                id: addressRequestedValidator
+                isRequested: root.isRequested
```

Comment:
As stated here: https://github.com/status-im/nim-status-client/pull/1231/files#r510841838 we still need to **Ignore** the `toValid` state inside tx preview to make it render the results we need.

I still haven't understood how us doing the detour through a validator is going to help, but changes are I'm missing something here...